### ISSUE TITLE
Green screen after closing game

### ISSUE THREAD

**Issue by canadahockey21** [uberhalit/EldenRingFpsUnlockAndMore#109]
Hello, I've been having an issue that happens about 80% of the time that I play ER. I will open through the application like normal, and can play fine. However, when I exit the game instead of going back to the desktop, I get a green screen that I can not get out of- so I end up manually turning off my PC and back on. I have uninstalled and reinstalled the app, but same thing. i5-7600k GTX 1080. Windows 10. Game version 1.03.2

**Comment by vicprox** [comment]
Is it [EldenRingFpsUnlockAndMore](https://github.com/uberhalit/EldenRingFpsUnlockAndMore) related? Does it crash to green screen if you don't start the game with the launcher?

**Comment by canadahockey21** [comment]
Yes, it's [EldenRingFpsUnlockAndMore](https://github.com/uberhalit/EldenRingFpsUnlockAndMore). If I open the game normally it closes without any issues- only when I use FPSunlock.

**Comment by vicprox** [comment]
Have you tried running the launcher with Admin rights?

**Comment by canadahockey21** [comment]
Yes I have, I always do.

**Comment by canadahockey21** [comment]
I actually took a video of this happening yesterday, do you want me to upload it to a site so you can take a look?
